hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientExecutionServiceImpl.java

```diff
@@ -20,14 +20,18 @@
 import com.hazelcast.core.ICompletableFuture;
```

Comment:
Not part of this PR; but is this class needed at all? 

So why do we need all these wrapper executors in the first place.